xcodeproj/internal/default_input_file_attributes_aspect.bzl

```diff
@@ -109,6 +109,7 @@ def _default_input_file_attributes_aspect_impl(target, ctx):
 
     return [
         XcodeProjAutomaticTargetProcessingInfo(
+            codesignopts = "codesignopts",
```

Comment:
We should only set this for rules that have it. I think that means the `AppleBundleInfo` and `AppleBinaryInfo` cases.